### PR TITLE
Monuments

### PR DESCRIPTION
**Pull request by OK-Diamond**
See Vanilla Repurpose section: https://docs.google.com/document/d/1MwzRb_F_Vmd3pkae3yGNPQnAi0-hcOsv7avHuaWrUxQ/edit

### CODE REVIEW

File: common/great_projects/pf_modified_monuments.txt [hunk 1]
```diff
@@ -0,0 +1,989 @@
...
+		area_modifier = {
+		}
+		country_modifiers = {
+			free_city_imperial_authority = 0.2
+			prestige = 1
+			tolerance_own = 1
+			church_loyalty_modifier = 0.15
+		}
+		on_upgraded = {
+            custom_tooltip = power_of_ulm
```
Comment (maxice8) [inline common/great_projects/pf_modified_monuments.txt:986]: This is not localised

**Review by maxice8 — CHANGES_REQUESTED** [review]

File: common/great_projects/pf_modified_monuments.txt (hunk 1 above)
Comment (OK-Diamond) [reply common/great_projects/pf_modified_monuments.txt:986]: oops, forgot to remove that
I'll update it and redo the pull req.

Comment (maxice8) [reply common/great_projects/pf_modified_monuments.txt:986]: No need to close, you can just create a new commit and push the same branch, the pull request will keep track of the changes

Comment (OK-Diamond) [reply common/great_projects/pf_modified_monuments.txt]: Thanks, didn't know this! I've now updated it.

**Review by Chewyshoot — APPROVED** [review]

**Review by maxice8 — APPROVED** [review]

**Pull request merged**